bessctl/cli.py

```diff
@@ -247,7 +259,7 @@ def _do_complete(self, line, partial_word):
         for cmd in self.cmdlist:
             syntax = cmd[0]
             match_type, sub_candidates, \
-                syntax_token, score = self.match(syntax, line)
+                syntax_token, _ = self.match(syntax, line)
```

Comment:
Doesn't `self.match` return a 5-tuple now?  If so there should be two `_`s (or use the new Python3 feature, `*_`).